prov/efa/src/efa_strerror.c

```diff
@@ -81,7 +81,14 @@ void efa_show_help(enum efa_errno err) {
 		help = "This error is detected locally. "
 		"The connection was previously established via handshake, "
 		"which indicates the error is likely due to the peer process no "
-		"longer being present.";
+		"longer being present, or a hardware failure happened on the remote peer.";
+		break;
+	case FI_EFA_ERR_UNESTABLISHED_RECV_UNRESP:
+		help = "This error is detected locally. "
+		"The peer is reachable by the EFA device but libfabric failed "
+		"to complete a handshake, which indicates the error is likely "
+		"due to the peer process no longer being present, "
+		"or a hardware failure happened on the remote peer.";
```

Comment:
Since Yehuda requested it, let's drop the HW failure here.
Sorry for the back and forth.